Add winget as Windows installation method for dbt Fusion (#8983)
fishtown-analytics/docs.getdbt.com: website/docs/guides/fusion-qs.md

    ```
    Or you can close and reopen your terminal window. This will load the updated environment settings into the new session.
</TabItem>
<TabItem value="windows-winget" label="Windows (winget)">

1. Run the following command in your terminal to install the <Constant name="fusion_engine" /> CLI:
    ```shell
    winget install dbt
    ```
2. To use `dbt` immediately after installation, reload your shell so that the new `Path` is recognized:
    ```powershell
    Start-Process powershell
    ```

>     Or you can close and reopen your terminal window. This will load the updated environment settings into the new session.

Reviewer: ```suggestion
    You can also close and reopen your shell window. This will load the updated environment settings into the new session.
```
